.eslintrc.js

```diff
@@ -96,7 +98,7 @@ module.exports = {
     plugins: ['@typescript-eslint', 'jsdoc', 'you-dont-need-lodash-underscore', 'react-native-a11y', 'react', 'testing-library'],
     parser: '@typescript-eslint/parser',
     parserOptions: {
-        project: './tsconfig.json',
+        project: path.resolve(__dirname, './tsconfig.json'),
```

Comment:
This change was really just a QOL improvement to fix [this issue](https://intellij-support.jetbrains.com/hc/en-us/community/posts/360009471420-parserOptions-project-eslint-path-is-resolved-wrong) with JetBrains IDE's.

inline lint warnings were not working for any files with a separate eslint config (tests, GitHub Actions, etc...)

This fixes that by using an absolute path for the project's `tsconfig.json`..